### ISSUE TITLE
ACC missing TCP Profile values after conversion

### ISSUE THREAD

**Issue by Maxximouss** [f5devcentral/f5-automation-config-converter#88]
### Is your feature request related to a problem? Please describe.
When I am converting configuration to AS3 declaration, I noticed that some elements are not supported by ACC Converter, like TCP Profiles inside virtual server configuration (tcp-f5-wan and tcp-f5-lan) 

Original app.conf :
```
ltm virtual /Common/vs_http {
    destination /Common/10.0.0.12:80
    ip-protocol tcp
    mask 255.255.255.255
    pool /Common/pool_http
    profiles {
        /Common/f5-tcp-lan {
            context serverside
        }
        /Common/f5-tcp-wan {
            context clientside
        }
        /Common/http { }
    }
    source 0.0.0.0/0
    translate-address enabled
    translate-port enabled
    vlans {
        /Common/internal
    }
    vlans-enabled
}
```

AS3 Converted Declaration : 
```
    "declaration": {
        "class": "ADC",
        "schemaVersion": "3.35.0",
        "id": "urn:uuid:c669cbf9-bcab-402d-8a16-6db7c2778de2",
        "label": "Converted Declaration",
        "remark": "Generated by Automation Config Converter",
        "Common": {
            "class": "Tenant",
            "Shared": {
                "class": "Application",
                "template": "shared",
                "vs_http": {
                    "layer4": "tcp",
                    "pool": "pool_http",
                    "translateServerAddress": true,
                    "translateServerPort": true,
                    "class": "Service_HTTP",
                    "profileHTTP": {
                        "bigip": "/Common/http"
                    },
                    "virtualAddresses": [
                        "10.0.0.1"
                    ],
                    "virtualPort": 80,
                    "snat": "none",
                    "allowVlans": [
                        {
                            "bigip": "/Common/internal"
                        }
                    ]
                },
```

### Describe the solution you'd like
I would like to obtain the AS3 declaration including all inputs elements like the TCP Profiles

**Comment by p-semenov-f5** [comment]
Looks like a bug:
2022-05-03 13:48:07 WARN Invalid reference dropped: /Common/f5-tcp-lan
2022-05-03 13:48:07 WARN Invalid reference dropped: /Common/f5-tcp-wan

Will investigate that.

**Comment by p-semenov-f5** [comment]
Fixed in 1.21

**Comment by Maxximouss** [comment]
Thanks for your help
